numbat/src/command.rs

```diff
@@ -0,0 +1,517 @@
+use std::str::SplitWhitespace;
+
+use crate::{
+    parser::ParseErrorKind,
+    span::{SourceCodePositition, Span},
+    ParseError,
+};
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum ListItems {
+    Functions,
+    Dimensions,
+    Variables,
+    Units,
+}
+
+enum ListAlias {
+    List,
+    Ls,
+}
```

Comment:
Not really the topic of your PR, but since we're cleaning up anyway: I think we should get rid of the `ls` alias. It's not in line with the rest of the commands. It's a two-character identifier that someone might want to use for a variable/unit name. And it complicates the code here.

Reply:
Done